app/Http/Controllers/Ledge/ManagementController.php

```diff
@@ -58,7 +53,7 @@ public function return($id)
                 'status' => LedgeStatus::AwaitingReturn
             ]);
 
-            Mail::to($ledge->lender->email)->send(new BookReturnStatusMail($ledge));
+            SendEmail::dispatch('App\Mail\BookReturnStatusMail', $ledge, $ledge->lender->email)->onQueue('mail');
```

Comment:
Due to the comment below (to change the queue to use default due to config issue) can you remove the `onQueue('mail')` to all this entry please :)

Reply:
Amended in 82764e4.